geps/gep-1713/index.md

```diff
@@ -434,11 +452,11 @@ metadata:
 spec:
   parentRefs:
   - name: second-workload-listeners
-    kind: ListenerSet
+    kind: XListenerSet
     sectionName: second
 ```
 
-For instance, the following `HTTPRoute` attempts to attach to a listener defined in the parent `Gateway` using the sectionName `foo`. This is not valid and the route's status `Accepted` condition should be set to `False`
+If routes MUST attach to a `XListenerSet` and its parent `Gateway`, it MUST have multiple `parentRefs` eg:
```

Comment:
Nit: "MUST" feels out of place here. This is meant to be used for instructions for implementers. Instead I'd reword this to

```suggestion
To attach a Route to a `XListenerSet` and its parent `Gateway`, it MUST have multiple `parentRefs` eg:
```

Reply:
fixed